package.json

```diff
@@ -91,15 +99,16 @@
     "trailingComma": "es5",
     "tabWidth": 2,
     "semi": false,
-    "singleQuote": true
+    "singleQuote": true,
+    "printWidth": 120
```

Comment:
The reason to go from 80 -> 120 was the way Prettier cuts lines without asking when using the pre-commit hook.

I prefer having line length rule on the Eslint side because...
1.  It gives just warning and doesn't force anything...
2. ...so I can fix it the way I want in a separate commit
3. ...so I can leave the long line if I really want and disable the warning with `/* eslint:disable max-len */`

Reply:
I added this as TODO item in `linter.md`

But for now  I prefer more flexible 120 line length.